app/views/articles/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Liste av alle blogginnleggene</h1>
```

Comment:
Here I see that text strings are hard coded into the template. This is fine for this task and for single-language projects. If you need to have multiple language support, this is built into Rails in the form of [I18n](http://guides.rubyonrails.org/i18n.html). You can check that out if you feel like it.